### PR TITLE
Remove unused tick-cluster.js, use version from ringpop-common

### PR DESCRIPTION
**Pull request by btromanova**
tick-cluster.js is used in tests. Currently local version was used, but since we want to keep only one instance of tick-cluster, it's better to do same thing as ringpop-node --- fetch ringpop-common and use tick-cluster from there.

### CODE REVIEW

**Comment by btromanova** [comment]
ringpop_common_branch should be changes to "master" from "tr-tick-cluster", once ringpop-common is merged.

File: package.json [hunk 1]
```diff
@@ -33,6 +33,7 @@
     "cli-table": "^0.3.1",
     "commander": "^2.6.0",
     "ringpop": "^10.0.0",
+    "farmhash": "^1.1.1",
```
Comment (dansimau) [inline package.json]: Why?

Comment (btromanova) [inline package.json]: tick-cluster uses it. Is there another way?

Comment (dansimau) [inline package.json]: Move along .... nothing to see here ....

**Comment by dansimau** [comment]
LGTM, once `run-tests` is pointed at ringpop-common master 👍

**Comment by jwolski** [comment]
did you consider submoduling ringpop-common instead of the fetch-ringpop-common() function?

**Comment by dansimau** [comment]
Using submodules was initially considered but in a slightly different case.

Fetching was initially done as part of the integration test runs, because fetching always ensures you have the latest tests.

For the test case, making ringpop-common a submodule means that ringpop-go and ringpop-node repos would each need to be updated every time there is a new test added (to receive the latest tests).

I wanted to avoid a situation where tests would accidentally not get run because someone forgot to update the commit the submodule was pointing to.

Also having to update the commit ref for submodules in each repo is more work. It would mean 3 PRs across 3 repos are required for every change to the integration tests.

In this PR, fetching tick-cluster this way may be less important. I'd reverse the question. What is the benefit of making it a submodule?

**Comment by btromanova** [comment]
I've rebased on master and added npm install in ringpop-common after the fetch step. This slows down tests a bit, possible follow-up would be to put tick-cluster to a separate package in ringpop-common to be able to install only its dependencies.

**Comment by jwolski** [comment]
@dansimau I'm mostly fine with either approach. I wanted to ask the question to understand whether submodules were considered... since the same thing (specifying the ringpop-common remote, specifying its branch, and pulling the latest submodule code on-demand) is achievable with them. I don't see why we'd need to immediately update ringpop-go and ringpop-node repos when ringpop-common is upgraded. Couldn't we just lazily update the submodule as we do with the "fetch approach?"

The benefit to the submodule approach that I see is that the submodule makes it more obvious that we have built/run ringpop-admin against a specific version of the integration test suite. Git will maintain a history of the reference to ringpop-common for us, whereas without the submodule it's unclear which version of ringpop-common should ringpop-admin be run against.

**Comment by dansimau** [comment]
> specifying the ringpop-common remote, specifying its branch, and pulling the latest submodule code on-demand

Oh wow. I didn't realise you can track branches now...

**Pull request merged**